aiohttp/web_request.py

```diff
@@ -605,8 +604,7 @@ def body_exists(self) -> bool:
                     out.add(field.name, value)
                     size += len(value)
                     if 0 < max_size < size:
-                        raise ValueError(
-                            'Maximum request body size exceeded')
+                        raise HTTPRequestEntityTooLarge
```

Comment:
May be add some message like what size was expected and which one is actual? That's good for debug.

Reply:
Makes sense.
We do it for too large headers already.

Reply:
Ok, so something like `Maximum request body size exceeded: 1234 > 123`?

Reply:
Or perhaps clearer `Maximum request body size 123 exceeded, actual body size 1234`?

Reply:
I like the second

Reply:
:+1:  the second. A bit verbose, but that wouldn't hurt.